dom/nodes/Node-appendChild-script-and-div-from-fragment.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>Node.appendChild: inserting script and div from a DocumentFragment</title>
+<link rel=help href="https://dom.spec.whatwg.org/#dom-node-appendChild">
+<link rel=help href="https://dom.spec.whatwg.org/#concept-node-insert">
+<link rel=help href="https://github.com/whatwg/dom/issues/575">
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<div id="log"></div>
+<script>
+var script = document.createElement("script");
+var div = document.createElement("div");
+
+var scriptRan = false;
+script.textContent = `
+    assert_equals(div.parentNode, script.parentNode, "div does not have same parent as script");
+    scriptRan = true;
+`;
+
+var df = document.createDocumentFragment();
+df.appendChild(script);
```

Comment:
And again...

Reply:
Added an assertion here and in other tests too.